source/github-helpers/get-user-avatar.ts

```diff
@@ -1,12 +1,18 @@
 import {$optional} from 'select-dom/strict.js';
 import * as pageDetect from 'github-url-detection';
 
-export default function getUserAvatar(username: string, size: number): string | void {
-	const cleanName = username.replace('[bot]', '');
+export default function getUserAvatar(username: string, size: number, href?: string): string | void {
```

Comment:
The function is already receiving the username. If it's wrong, it should be fixed *outside* this function.